IowaPD_people.yaml

```diff
@@ -389,20 +408,33 @@ entityDefinitions:
     - !<conditions.BooleanIsNullCondition>
       column: 'OfficerID'
       reverse: true
-  OfficersPersonDetails:
-    name: "OfficersPersonDetails"
-    fqn: 'ol.persondetails'
-    entitySetName: 'JohnsonCounty_IowaCity_JCJIPersonDetails2'
+  OfficersPhysicalAppearance:
```

Comment:
I am pretty sure this is not officers but other people. You have the same columns reading into the OthersPhysicalAppearance below, so we can just delete this whole entity set.

Reply:
this does get officers physical appearances, which are almost certainly always blank but I'll leave in because why not